readme.md

```diff
@@ -1,50 +1,59 @@
-# Setting Up Your Environment
+# gRPC Example
+
+## Setting Up Your Environment
+
 Keep in mind that gRPC requires golang v1.6 or greater, to install run
 
-```
+``` shell
```

Comment:
what does shell buy you here?

Reply:
It instructs markdown rendering engines about the langauge in the fenced code block. 

https://help.github.com/articles/creating-and-highlighting-code-blocks/